notebooks/PDF2Text.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "PDF2Text.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xZ4pCuLv6_OM"
+      },
+      "source": [
+        "### DEPENDENCIES:\r\n",
+        "\r\n",
+        "• pdf2image  \r\n",
+        "    `converts each page in a pdf file into a jpeg image`\r\n",
+        "\r\n",
+        "• pytesseract  \r\n",
+        "    `converts the jpeg images into text`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "BV_69zzv2woS"
+      },
+      "source": [
+        "%%capture\r\n",
+        "!sudo apt-get install tesseract-ocr\r\n",
+        "!sudo apt-get install poppler-utils --fix-missing\r\n",
+        "!pip install pytesseract\r\n",
+        "!pip install pdf2image"
+      ],
+      "execution_count": 1,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "_1LMVGiaCDTj",
+        "outputId": "7d75aa1a-b4fd-46a2-c317-62d4ba182f1e"
+      },
+      "source": [
+        "from typing import List, Tuple\r\n",
+        "import pytesseract \r\n",
+        "from pdf2image import convert_from_path \r\n",
+        "import os\r\n",
+        "\r\n",
+        "from google.colab import drive\r\n",
+        "drive.mount('/content/drive')"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Mounted at /content/drive\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LM3Y_uBp70x2"
+      },
+      "source": [
+        "• This code uses google drive in order to host the files. If using a different file structure, set your own paths variables `PDF_DIR` and `TXT_DIR`"
+      ]
+    },
```

Comment:
Is hosting the files in google drive a temporary measure?  If so, can we leave more explicit documentation about alternatives?

Reply:
Completely temporary. I'm imagining in the future, once a user uploads their pdf, the conversion gets run, the txt gets scraped, and that info gets dumped into the database.